src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -706,9 +838,13 @@ class DistributedClosestPointImpl
     bvh->setAllocatorID(m_allocatorID);
```

Comment:
Please remove an outdated comment a few lines up 
```
/// GOT TO HERE -- fix for templated ExecSpace!
```
(I'm sure it meant something when I added it :) )

Reply:
No problem!